### ISSUE TITLE
run error

### ISSUE THREAD

**Issue by fserdogan** [yilmaznaslan/berlin-auslaenderbehorde-termin-bot#25]
Hello, 

I get the below error regularly after 3-4 hours of running the code. Do you have any idea about the reason?

`INFO IoUtils File name :Section1MainPageHandler_2023-04-10_23:21:50_exception.html, Section1MainPageHandler_2023-04-10_23:21:50_exception.png`

**Comment by yilmaznaslan** [comment]
Yeah thats normal. Sometimes the page doesn't load in the given time or sometimes the webpage of the auslandebehorde is down for several minutes. You see such logs in such situations. Once the auslaenderbehorde homepage is up again, it works normal

**Comment by yilmaznaslan** [comment]
p.s: you should be able to find the file **Section1MainPageHandler_2023-04-10_23:21:50_exception.png** in the root directory where you run the **.gradlew run**. You can see the webpage at that moment

**Comment by fserdogan** [comment]
Understood, thanks for the explanation. Thanks for the bot btw, it makes a great help. And if doable, an improvement suggestion would be making the bot continue when the webpage available again. Because as I tested since a few days, bot stops with the error and needs to be restarted manually.

**Comment by fserdogan** [comment]
I pulled the latest version and when I run "./gradlew run" I got an error as below. I tried running again an earlier version and this error doesn't show up. Is it an error on my side or the code is changed?

`Starting a Gradle Daemon (subsequent builds will be faster)

FAILURE: Build failed with an exception.

* Where:
Build file '/Users/g/Desktop/berlin_bot/build.gradle'

* What went wrong:
Could not compile build file '/Users/g/Desktop/berlin_bot/build.gradle'.
> startup failed:
  General error during conversion: Unsupported class file major version 64
  
  java.lang.IllegalArgumentException: Unsupported class file major version 64`
